plugin-ci/orb.yml

```diff
@@ -104,11 +104,22 @@ jobs:
       - checkout
       - *restore_cache
       - run: make dist
+      - run:
+          name: "Generating release notes"
+          command: |
+            echo "### Supported Mattermost Server Versions: $(cat plugin.json | jq .min_server_version -r)+"  >> release-notes.md
+            if [[ $(git tag -l | wc -l) -eq 1 ]]; then
+              git log --pretty=oneline --abbrev-commit --no-decorate --no-color $(git rev-list --max-parents=0 HEAD) HEAD >> release-notes.md;
+            else
+              git log --pretty=oneline --abbrev-commit --no-decorate --no-color tags/$(git describe --tags --abbrev=0)^..HEAD >> release-notes.md;
+            fi
+            mv release-notes.md dist
```

Comment:
Why not output directly to `dist/release-notes.md`?

Reply:
Agree.